ingestion_tools/scripts/standardize_dirs.py

```diff
@@ -58,6 +59,7 @@
                 FrameImporter: {},
                 TiltSeriesImporter: {},
                 RawTiltImporter: {},
+                CollectionMetadataImporter: {},
```

Comment:
This also needs to be added to https://github.com/chanzuckerberg/cryoet-data-portal-backend/pull/257/files#diff-a112fce9544e4d2a0ebafa1a467c88c2ba702ec321b75651e92683d9fd319d6bL24 for it to support the relevant import flags.

Reply:
can you elaborate. I'm not sure what you mean by this?

Reply:
For us to have an `--import-collection-metadata` flag that is supported by ingestion process, we need to have this importer class listed in the `IMPORTERS` constant.